pkg/chunked/cache_linux.go

```diff
@@ -465,17 +600,20 @@ func writeCache(manifest []byte, format graphdriver.DifferOutputFormat, id strin
 	logrus.Debugf("Written lookaside cache for layer %q with length %v", id, counter.Count)
 
 	return &cacheFile{
-		digestLen: digestLen,
-		tagLen:    tagLen,
-		tags:      tagsBuffer.Bytes(),
-		vdata:     vdata.Bytes(),
+		digestLen:   digestLen,
+		tagLen:      tagLen,
+		tags:        tagsBuffer.Bytes(),
+		vdata:       vdata.Bytes(),
+		fnames:      fnames.Bytes(),
+		fnamesLen:   len(fnames.Bytes()),
+		bloomFilter: bloomFilterFromTags(tags, digestLen),
```

Comment:
This was already computed inside `writeCacheFileToWriter`; couldn’t that happen only once?